multimedia/tvheadend/Makefile

```diff
@@ -70,8 +70,12 @@ CONFIGURE_ARGS += \
 	--arch=$(ARCH) \
 	--disable-dbus_1 \
 	--disable-libav \
+	--disable-ffmpeg_static \
```

Comment:
This change should be described in the commit message, because you are doing more changes than just version bump.